src/Akeneo/Tool/Bundle/ElasticsearchBundle/Client.php

```diff
@@ -378,13 +386,27 @@ public function getConfigurationLoader(): Loader
 
     /**
      * @param array $body an array containing a query compatible with https://www.elastic.co/guide/en/elasticsearch/reference/5.5/docs-delete-by-query.html
+     * @throws Conflict409Exception
      */
     public function deleteByQuery(array $body): void
     {
-        $this->client->deleteByQuery([
-            'index' => $this->indexName,
-            'body' => $body,
-        ]);
+        $attempts = 0;
+        $exception = null;
+        do {
+            $attempts++;
+            try {
+                $this->client->deleteByQuery([
+                    'index' => $this->indexName,
+                    'body' => $body,
+                ]);
+                return;
+            } catch (Conflict409Exception $e) {
+                $exception = $e;
+                sleep($this->maxExpectedIndexationLatency);
```

Comment:
```suggestion
                usleep($this->maxExpectedIndexationLatency);
```
Sleep parameter is in seconds. 100 seconds is huge. I guess you wanted 100 microsecs.

Reply:
I convert  it in constructor... I wanted milliseconds...